packages/client/src/components/Oauth/Oauth.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect } from 'react'
+import { useGetOAuthYandexServiceIdMutation } from '../../store/auth/auth.api'
+import { selectServiceId } from '../../store/user/userSlice'
+import { useAppSelector } from '@/store'
+
+const REDIRECT_URI = 'http://localhost:3000'
+
+const renderOAuthLink = (service_id: string, redirect_uri: string): string => {
+  if (typeof service_id !== 'string') {
+    return '#'
+  }
+
+  return `https://oauth.yandex.ru/authorize?response_type=code&client_id=${service_id}&redirect_uri=${redirect_uri}`
+}
+
+const OAuthPanel: React.FC = () => {
+  const service_id = useAppSelector(selectServiceId)
```

Comment:
https://github.com/imozi/Guardian-of-the-Galaxy/pull/47/files#r1104154544

Reply:
Done